openedx/core/lib/teams_config.py

```diff
@@ -190,7 +210,7 @@ def __repr__(self):
         """
         Return developer-helpful string.
         """
-        attrs = ['teamset_id', 'name', 'description', 'max_team_size', 'teamset_type']
+        attrs = ['teamset_id', 'name', 'description', 'max_team_size', 'teamset_type', 'dynamic_user_partition_id']
```

Comment:
```suggestion
        attrs = ['teamset_id', 'name', 'description', 'max_team_size', 'teamset_type', 'user_partition_id']
```

There is no such thing as a static team-set partition, so in the spirit of brevity, you could just call this `user_partition_id`.

I know that'd be a big change at this point, so feel free to skip this suggestion.

Reply:
thanks for doing this 😄